frontend/packages/policy-editor/src/components/SecurityLevel/SecurityLevel.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode } from 'react';
+import classes from './SecurityLevel.module.css';
+import { Select, Label, Paragraph, HelpText, Link } from '@digdir/design-system-react';
+import { useTranslation, Trans } from 'react-i18next';
+import { RequiredAuthLevel } from '../../types';
+
+const SELECT_AUTH_LEVEL_ID: string = 'select-auth-level';
+const URL_TO_SECURITY_LEVEL_PAGE: string =
+  'https://info.altinn.no/hjelp/innlogging/diverse-om-innlogging/hva-er-sikkerhetsniva/';
+
+export const authlevelOptions = [
+  { value: '0', label: 'policy_editor.auth_level_option_0' },
+  { value: '1', label: 'policy_editor.auth_level_option_1' },
+  { value: '2', label: 'policy_editor.auth_level_option_2' },
+  { value: '3', label: 'policy_editor.auth_level_option_3' },
+  { value: '4', label: 'policy_editor.auth_level_option_4' },
+];
+
+export type SecurityLevelProps = {
+  requiredAuthenticationLevelEndUser: RequiredAuthLevel;
+  onSave: (authLevel: RequiredAuthLevel) => void;
+};
+
+export const SecurityLevel = ({
+  requiredAuthenticationLevelEndUser,
+  onSave,
+}: SecurityLevelProps): ReactNode => {
+  const { t } = useTranslation();
+
+  const authLevelOptionKeysAsDisplayStrings = authlevelOptions.map((option) => ({
+    ...option,
+    label: t(option.label),
+  }));
+
+  return (
+    <div>
+      <Label as='p' size='medium' spacing>
+        {t('policy_editor.security_level_label')}
+      </Label>
```

Comment:
`Label` må være direkte knyttet til et inndatafelt. Her gir det kanskje mer mening å bruke [`Heading`](https://storybook.designsystemet.no/?path=/docs/felles-typography-heading--docs)?

Reply:
Er derfor jeg har lagt på as='p' på den, for å slippe å ha den knyttet til et inndatafelt. Men kan godt gjøre den om til heading ja :)

Reply:
Den overså jeg faktisk, men jeg tenker likevel `Heading` er bedre, så unngår vi misforståelser :-)